### DISCUSSION TITLE
Any way to “reverse” having committed the “reverse()” common pitfall?

### DISCUSSION THREAD

**Discussion by brycewray** [11ty/eleventy#1756] [Q&A] [answered]
One of the [common pitfalls](https://www.11ty.dev/docs/pitfalls/) is:

> My collections are out of order because I’m using `Array reverse()` on the collections global.

If you have done this at some point before knowing it was a [no-no](https://www.11ty.dev/docs/collections/#array-reverse), is there a way to “clear” the collections and thus start over again? And is there a recommended setting within [`DEBUG`](https://www.11ty.dev/docs/debugging/) that would allow diagnosing this?

**Comment by Aankhen** [comment, marked as answer]
We resolved this on Discord, and I thought I’d repeat it here in case anyone else has the same question: this is not a permanent change to your collections, since they’re regenerated each time you build your site, so if you change your code to operate on copies instead of the original collections, you should see the correct order in the next build.
